Orange/regression/tree.py

```diff
@@ -17,6 +17,24 @@ class TreeRegressionLearner(SklLearner):
                      SklImpute(),
                      Continuize()]
 
+    """
+    criterion = The function to measure the quality of a split.
```

Comment:
Have the docstrings be the first thing in the class definition block. And let them start with a brief sentence of introduction, [Napoleon-style](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html).